index.html

```diff
@@ -38,6 +38,107 @@ <h2 class="connect">LET'S CONNECT</h2>
+                    <h3 class="project-name">Tonic</h3>
+                        <div class="key-text"> 
+                            <p class="canopy">CANOPY</p>
+                            <p class="fa-solid fa-circle fa-2xs"></p>
+                            <p class="position">Back end dev</p>
+                            <p class="fa-solid fa-circle fa-2xs"></p>
+                            <p class="year">2015</p>
+                        </div>
+                        <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <ul class="language-container">
+                            <li class="lang">html</li>
+                            <li class="lang">css</li>
+                            <li class="lang">javascript</li>
+                        </ul>
+                        <button class="see-project">See Project</button>
+                </div>            
+            </div>
+        </section>
+
+        <section class="about">
+            <h1 class="name">About Myself</h1>
+            <p class="description">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <h2 class="connect">LET'S CONNECT</h2>
+            <ul class="social-icons">
+                <li class="icon"><a class="fa-brands fa-twitter" href="#"></a> </li>
+                <li class="icon"><a class="fa-brands fa-linkedin" href="#"></a> </li>
+                <li class="icon"><a class="fa-brands fa-medium" href="#"></a> </li>
+                <li class="icon"><a class="fa-brands fa-github" href="#"></a> </li>
+                <li class="icon"><a class="fa-brands fa-angellist" href="#"></a> </li>
+            </ul>
+            <button class="see-resume">Get my Resume</button>
+
+            <div class="skills-container">
+                <div class="languages">
+                    <div class="text-icon"> 
+                    <h2 class="skill-type">Languages</h2>
+                    <a class="drop-down">
+                        <i class="fa-solid fa-angle-down"></i>
+                    </a>
+                    </div>
+                    <div class="about-languages-container">
+                        <ul class="about-language"><li><img src="file:///C:/All%20stuff/Microverse/My-Portfolio/images/javascript.svg" alt="javascript"><p>JAVASCRIPT</p></li></ul>
+                        <ul class="about-language"><li><img src="file:///C:/All%20stuff/Microverse/My-Portfolio/images/Html.svg" alt="html"><p>HTML</p></li></ul>
+                        <ul class="about-language"><li><img src="file:///C:/All%20stuff/Microverse/My-Portfolio/images/CSS.svg" alt="css"><p>CSS</p></li></ul>
```

Comment:
All the same, thing applies here for the image please give the correct image path for all of them. Currently, it is looking like this: 

<img width="290" alt="image" src="https://user-images.githubusercontent.com/34578631/159097221-384314ad-fc59-413f-801b-848cf6ca46ba.png">

Reply:
In Figma design, the images are aligned vertically with the image, I have hit a blocker on trying to do that. I have fixed the other errors, but can't figure out how to align images with text vertically. If you could help me with that, I would appreciate that.
@Mre55